folio/reports/missing_items.sql

```diff
@@ -0,0 +1,40 @@
+-- metadb:function missing_items
+
+-- Report pulls a list of missing items, by location if entered, for searching.
+
+DROP FUNCTION IF EXISTS missing_items;
+
+CREATE FUNCTION missing_items
```

Comment:
First of all, if the function has no parameters, this line should be
```
CREATE FUNCTION missing_items()
```

That is not the only error.

The function should be tested before submitting a pull request.